doc/run_experiment.rst

```diff
@@ -633,219 +698,185 @@ The number of features used by the `FeatureHasher <https://scikit-learn.org/stab
-will be munged together to make names.
+.. _ids_to_floats:
 
-.. _fixed_parameters:
+ids_to_floats *(Optional)*
+""""""""""""""""""""""""""
 
-fixed_parameters *(Optional)*
-"""""""""""""""""""""""""""""
+If you have a dataset with lots of examples, and your input files have IDs that
+look like numbers (can be converted by float()), then setting this to True will
+save you some memory by storing IDs as floats. Note that this will cause IDs to
+be printed as floats in prediction files (e.g., ``4.0`` instead of ``4`` or
+``0004`` or ``4.000``).
 
-List of dicts containing parameters you want to have fixed for each
-learner in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used). If :ref:`grid_search` is ``True``,
-there is a potential for conflict with specified/default parameter grids
-and fixed parameters.
+.. _label_col:
 
-The default fixed parameters (beyond those that scikit-learn sets) are:
+label_col *(Optional)*
+""""""""""""""""""""""
 
-AdaBoostClassifier and AdaBoostRegressor
-    .. code-block:: python
+If you're using :ref:`ARFF <arff>`, :ref:`CSV <csv>`, or :ref:`TSV <csv>`
+files, the class labels for each instance are assumed to be in a column with
+this name. If no column with this name is found, the data is assumed to be
+unlabelled. Defaults to ``y``. For ARFF files only, this must also be the final
+column to count as the label (for compatibility with Weka).
 
-       {'n_estimators': 500, 'random_state': 123456789}
+.. _learning_curve_cv_folds_list:
 
-DecisionTreeClassifier and DecisionTreeRegressor
-    .. code-block:: python
+learning_curve_cv_folds_list *(Optional)*
+""""""""""""""""""""""""""""""""""""""""""
 
-       {'random_state': 123456789}
+List of integers specifying the number of folds to use for cross-validation
+at each point of the learning curve (training size), one per learner. For
+example, specifying ``["SVC", "LogisticRegression"]`` for ``learners``
+and  specifying ``[10, 100]`` for ``learning_curve_cv_folds_list`` will
+tell SKLL to use 10 cross-validation folds at each point of the SVC curve and
+100 cross-validation folds at each point of the logistic regression curve. Although
+more folds will generally yield more reliable results, smaller number of folds
+may be better for learners that are slow to train. Defaults to 10 for
+each learner.
 
-DummyClassifier
-    .. code-block:: python
+.. _learning_curve_train_sizes:
 
-       {'random_state': 123456789}
+learning_curve_train_sizes *(Optional)*
+""""""""""""""""""""""""""""""""""""""""""
 
-ElasticNet
-    .. code-block:: python
+List of floats or integers representing relative or absolute numbers
+of training examples that will be used to generate the learning curve
+respectively. If the type is float, it is regarded as a fraction of
+the maximum size of the training set (that is determined by the selected
+validation method), i.e. it has to be within (0, 1]. Otherwise it is
```

Comment:
```suggestion
validation method), i.e. it has to be within [0, 1]. Otherwise it is
```

Reply:
I actually intended to use the left open interval since you can't really have 0 as the train size for which to plot the learning curve :)